convert_hf_to_gguf.py

```diff
@@ -1497,6 +1497,17 @@ def get_audio_config(self) -> dict[str, Any] | None:
     def set_type(self):
         self.gguf_writer.add_type(gguf.GGUFType.MMPROJ)
 
+    def prepare_metadata(self, vocab_only: bool):
+        super().prepare_metadata(vocab_only)
```

Comment:
```suggestion
        super().prepare_metadata(vocab_only=vocab_only)
```
Might as well be `False`, but let's keep it like this in case it becomes useful in the future.